### ISSUE TITLE
Bottom to top transition?

### ISSUE THREAD

**Issue by aksonov** [ericvicenti/navigation-rfc#47]
i can't find how to make this transition with NavigationCard. Example shows left-to-right only...

**Comment by ericvicenti** [comment]
@hedgerwang has been working on this. Hedger, maybe you can write a quick note about it here that can be pasted into a doc later? Or better yet, just edit the doc in this repo and link to it?

**Comment by aksonov** [comment]
Any update on this?

**Comment by ericvicenti** [comment]
It is supported by the card stack:

```
<NavigationCardStack
  direction="vertical"
/>
```

You can look into the implementation of `NavigationCardStack` to see how to configure a vertical `NavigationCard`

**Comment by hasancanozkan** [comment]
Hi! is there any way to implement the transition from top to bottom? @ericvicenti

**Comment by ericvicenti** [comment]
@hasancanozkan, nope. But it wouldn't be too difficult to hack that into react-navigation
